wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,453 @@
+    - Like his predecessor, he continued working with the osu!team to manage the mapping/modding scenes. <!-- source is discord logs -->
+- **2018-12-17:** The QAT was declared as having no future in its current state and ideas for reform were encouraged.
+    - This decision was made because the QAT had not been involved in anything Quality Assurance related for years, however a system needed to exist to maintain the Beatmap Nominators and mapping community in general.
+    - Some concerns at the time about the QAT included:
+        - Little QAT engagement in controversial maps.
+        - Limited accessibility for new users to enter the team, causing stagnation.
+        - The role of the QAT in the mapping community was unclear.<!-- https://osu.ppy.sh/community/forums/topics/842601 -->
+
+**2749 beatmaps** were ranked in 2018. This year's impressive growth can be largely attributed to the QAT's discontinued Quality Assurance role and higher initiative from Beatmap Nominators.
+
+## 2019
+
+### January
+
+- **2019-01-29:** A proposal for QAT reform was created as a conglomeration of other users' proposals.
+    - The [Nomination Assessment Team (*NAT*)](/wiki/People/The_Team/Nomination_Assessment_Team) was introduced through this proposal, and was implemented later this year. <!-- https://osu.ppy.sh/community/forums/topics/862002 -->
+
+### March
+
+- **2019-03-01:** The [Mappers' Guild](/wiki/Mappers_Guild) transitioned from application-based entry to automatic entry for anyone with 3 ranked maps.
+    - [Featured Artist](/wiki/Featured_Artists) beatmaps became more frequently created and ranked.
+    - `#mappers-guild` channel was created in the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+- **2019-03-13:** Rule requiring consecutive nominations to be set at least 24 hours apart from each other was removed.
+    - This rule was a major complaint from mappers and Beatmap Nominators because it felt like it was unnecessarily wasting everyone's time. If an issue was found after the first nomination, the second nominator would often avoid pointing it out to avoid an extra 24 hour wait.
+    - Along with this change, Beatmap Nominators could veto qualified beatmaps. The rule originally existed because Beatmap Nominators needed 24 hours to potentially apply vetoes to maps with one nomination.<!-- https://osu.ppy.sh/community/forums/posts/6885033 -->
+
+### May
+
+- **2019-05-05:** The BN/QAT restructure was finalized.
+    - The Quality Assurance Team became the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
+    - Beatmap Nominators became responsible for disqualifications and veto mediation.
+    - The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks. <!-- https://osu.ppy.sh/home/news/2019-05-05-bn-and-qat-restructure -->
```

Comment:
At this time, BN applications changed from being cycle based to allowing users to apply at any time, with individualized cooldown periods if they did not get in or were removed. This is a pretty big shift from the previous large and occasional waves being replaced with new bns being added in small amounts more regularly.

Besides saving the NAT from spreadsheet insanity, I think that's one of the bigger changes the website offered for the community view.